mhw_analysis/ai/ai_mhw_images.py

```diff
@@ -0,0 +1,295 @@
+# imports
+from importlib import reload
+import os
+import numpy as np
+from matplotlib import pyplot as plt
+
+import iris
+
+import sys
+
+# create module paths
+MHW_PATH = '/Users/kamil/Desktop/research2020/mhw_analysis'
```

Comment:
This is the Repository path, right?
You can access it with:

```
from pkg_resources import resource_filename
resource_filename('mhw_analysis', SUB_FOLDER)
```

where SUB_FOLDER is any sub-folder in the repo.

Please do that.

Same for `OCEANPY_PATH`.

No hard-coding paths.